src/plugins/login/loginPlugin.ts

```diff
@@ -28,7 +33,8 @@ export class AzureLoginPlugin {
     try {
       if (subscriptionId && clientId && secret && tenantId) {
         authResult = await loginWithServicePrincipalSecretWithAuthResponse(clientId, secret, tenantId);
-      } else {
+      }
```

Comment:
nit: Move back to same line as `}`. I know I moved this when I commented that section out